cypress/e2e/topics/createEditDeleteTopic.feature

```diff
@@ -41,84 +41,84 @@ Feature: Topics - To create, edit and delete topics by the teacher.
-    When I enter description 'Cy this is the description of the topic. It is used for automated Cypress tests.' into element Text in element position '0'
+    When I enter title 'CypressAut Title for Text Element in Topic' into element Text in element position '0'
+    When I enter description 'CypressAut this is the description of the topic. It is used for automated Cypress tests.' into element Text in element position '0'
     When I click on button Add GeoGebra to topic
-    When I enter title "Cy Title for GeoGebra Element in Topic" into element GeoGebra
+    When I enter title "CypressAut Title for GeoGebra Element in Topic" into element GeoGebra
     When I enter GeoGebra material ID 'kEBfU7AR'
     When I click on button Add Learning Material to topic
-    When I enter title 'Cy Title for Learning Material Element in Topic' into element Learning Material
+    When I enter title 'CypressAut Title for Learning Material Element in Topic' into element Learning Material
     Then I see second learning material button in the content area
     # Note: currently step for adding material is excluded because this process is via new browser window
     When I click on button Add Etherpad to topic
-    When I enter title 'Cy Title for Etherpad Element in Topic' into element Etherpad in element position '3'
+    When I enter title 'CypressAut Title for Etherpad Element in Topic' into element Etherpad in element position '3'
     When I enter description for the ether pad 'this is my epad description' in element position '3'
     When I click on button Add Task to topic
-    When I enter title 'Cy Title for Task Element in Topic' into element Task
+    When I enter title 'CypressAut Title for Task Element in Topic' into element Task
     When I enter URL of the task from the another course for task id '59cce3f6c6abf042248e888d'
     When I click on create button to create topic
-    Then I can see edit topic page 'Cy Topic Creating and Deleting Test'
+    Then I can see edit topic page 'CypressAut Topic Creating and Deleting Test'
     When I click on save button to save changes
-    Then I see topic detail page "Cy Topic Creating and Deleting Test" with content elements "Cy Title for Text Element in Topic", "Cy Title for GeoGebra Element in Topic", "Cy Title for Learning Material Element in Topic", "Cy Title for Etherpad Element in Topic" and "Cy Title for Task Element in Topic"
-    When I click on last breadcrump element in topbar navigation
-    Then I can see topic 'Cy Topic Creating and Deleting Test' on course page
+    Then I see topic detail page "CypressAut Topic Creating and Deleting Test" with content elements "CypressAut Title for Text Element in Topic", "CypressAut Title for GeoGebra Element in Topic", "CypressAut Title for Learning Material Element in Topic", "CypressAut Title for Etherpad Element in Topic" and "CypressAut Title for Task Element in Topic"
+    When I navigate back to topic via breadcrump menu
```

Comment:
should this be 'navigate back to course detail...'?

Reply:
yes, you're right